src/helpers/runNotificationListener.ts

```diff
@@ -34,13 +35,18 @@ getFeedsContract.on(
   'CommentAdded',
   async (feedId: BigNumber, postId: BigNumber, commentId: BigNumber) => {
     try {
-      const tokens = await getTokens({ repliesEnabled: true })
-      await sendFirebaseNotification({ tokens })
-
-      if (commentId.toNumber() !== minimumNumberOfComments) return
       const numberFeedId = feedId.toNumber()
       const numberPostId = postId.toNumber()
+      const numberCommentId = commentId.toNumber()
+      if (
+        (await isBanned(numberFeedId, numberPostId)) ||
+        (await isBanned(numberFeedId, numberPostId, numberCommentId))
```

Comment:
I don't think the second condition will be true because it won't and cannot be banned when the listener catches a new comment.
I assume we need to check only the post, if it is banned or not.

Reply:
If you want to check the parent comment (replyTo), you have a fourth prop (eventComment) which contains `replyTo`. If it's equal to 0, this is a comment to the post; otherwise, it's a comment to comment.

Reply:
Let's remove this check for now - `await isBanned(numberFeedId, numberPostId, numberCommentId)`
Don't forget to create a task to check parent comment if it's block, before notify about it